src/pivot-table/components/cells/styles.tsx

```diff
@@ -68,3 +68,34 @@ export const StyledLabel = styled("div", {
   fontSize,
   fontFamily,
 }));
+
+export const AdjusterHitArea = styled(Box, {
```

Comment:
Any value in having this being a `Box` instead of just `"div"`?

Reply:
nope, just copy pasta from sn-table. Maybe should be div for consistency in the repo